app/dashboard/views.py

```diff
@@ -2501,6 +2519,31 @@ def invalid_file_response(uploaded_file, supported):
 
     return response
 
+@csrf_exempt
+@require_POST
+def bounty_upload_nda(request):
```

Comment:
We removed this functionality recently, not sure why its showing up in this PR - can you please remove?